truthbeam/internal/metadata/testdata/config.yaml

```diff
@@ -12,3 +12,9 @@ client_config:
   tls:
     insecure: true
     insecure_skip_verify: true
+# Cache TTL for compliance metadata
+# Options:
+# - 1m, 5m, 10m, 30m, 1h, 6h, 12h, 24h, 72h, 168h (duration format)
+# - 0 or -1ns (no expiration - cache forever) - DEFAULT
```

Comment:
Is `-1ns` correct or is a typo?

Reply:
Typo. Thanks @marcusburghardt . I think anything under zero will be seen as no expiration by the library, but docs should just say zero.